src/test/run-rbd-tests

```diff
@@ -41,7 +50,7 @@ run_generic_tests() {
 
 run_api_tests() {
     # skip many_snaps since it takes several minutes
-    nosetests -v test_rbd -e '.*many_snaps'
+    $NOSETESTS -v test_rbd -e '.*many_snaps'
```

Comment:
fwiw i was changing these all to do python3 -m nose ... which seems to work?

Reply:
changed and repushed.